src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/FlowLayoutPanelDesigner .cs

```diff
@@ -637,7 +637,7 @@ private void ReorderControls(DragEventArgs de)
             else
             {
                 // We are doing a copy, so let's copy the controls.
-                ArrayList tempList = new ArrayList();
+                List<Control> tempList = new();
                 tempList.AddRange(_dragControls);
 
                 DesignerUtils.CopyDragObjects(tempList, Component.Site);
```

Comment:
Shouldn't this be 
```suggestion
                tempList = DesignerUtils.CopyDragObjects(tempList, Component.Site);
```
?

Reply:
Hmm.. This does look a bit suspicious, but I am concerned about whether this was intentional and what effects this would have if we do make this change. It's not immediately clear from looking here. Let's open a separate issue on this to investigate.